src/Listener/CodeCoverageListener.php

```diff
@@ -150,22 +147,36 @@ public function beforeSuite(SuiteEvent $event): void
 
         $filter = $this->coverage->filter();
 
-        foreach ($this->options['whitelist'] as $option) {
-            $settings = $this->filterDirectoryParams($option);
-
-            $filter->includeDirectory($settings['directory'], $settings['suffix'], $settings['prefix']);
-        }
-
+        //We compute the list of file / folder to be excluded
+        //If the blacklist contains suffixes and/or prefixes, we extract an
+        //exhaustive list of files that match to be added in the excluded list.
```

Comment:
Personal opinion: I would put a whitespace between `//` and the text.

Reply:
Ok, I've pushed the change.